spec/features/index_page_spec.rb

```diff
@@ -103,4 +104,29 @@
       expect(page).to have_content('Page data: Past year')
     end
   end
+
+  context 'filter by document_type' do
+    before do
+      content_data_api_has_content_items(from: from, to: to, organisation_id: 'org-id', document_type: 'news_story', items: [items.first])
+      select 'News story', from: 'document_type'
+      click_on 'Filter'
+    end
+
+    it 'selects the document_type in the dropdown menu' do
+      expect(page).to have_select('document_type', selected: 'News story')
+    end
+
+    it 'renders the filtered results' do
+      table_rows = extract_table_content('.content-table table')
+      expect(table_rows.count).to eq(2)
```

Comment:
Can we more explicity check that our items shown in the table are all of the correct `Content Type`?

Reply:
Would like to only use one expect per example - use the `all` matcher we can remove counting the rows and make our test less fragile.
```
_header = table_rows.shift
expect(table_rows).to all(inlcude('News story'))
```